File: MiniDOMAsteriskLib/CallManagers/Events/MeetmeLeave.vb
Imports System.Net.Sockets
Imports System.Threading
Imports System.Net

Namespace CallManagers.Events

    
    Public Class MeetmeLeave
        Inherits AsteriskEvent

        Public Sub New()
            MyBase.New("MeetmeLeave")
        End Sub

        Public Sub New(ByVal rows() As String)
            MyBase.New(rows)
        End Sub

        Public ReadOnly Property Channel As String
            Get
                Return Me.GetAttribute("Channel")
            End Get
        End Property

        Public ReadOnly Property Meetme As String
            Get
                Return Me.GetAttribute("Meetme")
            End Get
        End Property

        Public ReadOnly Property Usernum As String
            Get
                Return Me.GetAttribute("Usernum")
            End Get
        End Property

        Public ReadOnly Property UniqueID As String
            Get
                Return Me.GetAttribute("UniqueID")
            End Get
        End Property
         
    End Class

End Namespace

File: MiniDOMAsteriskLib/CallManagers/Events/MeetmeLeave.cs

namespace minidom.CallManagers.Events
{
    public class MeetmeLeave : AsteriskEvent
    {
        public MeetmeLeave() : base("MeetmeLeave")
        {
        }

        public MeetmeLeave(string[] rows) : base(rows)
        {
        }

        public string Channel
        {
            get
            {
                return GetAttribute("Channel");
            }
        }

        public string Meetme
        {
            get
            {
                return GetAttribute("Meetme");
            }
        }

        public string Usernum
        {
            get
            {
                return GetAttribute("Usernum");
            }
        }

        public string UniqueID
        {
            get
            {
                return GetAttribute("UniqueID");
            }
        }
    }
}